vars/githubscm.groovy

```diff
@@ -74,6 +74,18 @@ def mergeSourceIntoTarget(String repository, String sourceAuthor, String sourceB
     """
 }
 
+def mergePR(String PRLink, String credentialID='kie-ci') {
```

Comment:
`PRLink` should be `pullRequestLink` (or something similar)

Reply:
changed